### DISCUSSION TITLE
솔루션 IP 지정에 대해 문의드립니다.

### DISCUSSION THREAD

**Discussion by ebutmanager** [commerce-api-naver/commerce-solutions#91] [묻고 답하기]
안녕하세요. 
커머스솔루션에 등록되어 있는 '이벗매니저'라고 합니다.
며칠전부터 '커머스API 모니터링 탐지 안내' 라는 메일이 옵니다.

커머스API센터에 로그인을 해보면 애플리케이션 아이디가 다르게 나오네요. 
어떻게 수정할까요?

1. 현재 로그인하면 확인할 수 있는 정보
- 아이디 : 
- 애플리케이션 아이디 : 4QOxzzyeiW6************
- 사업자정보 : 759-81-01283

2. 모니터링에 의해 탐지된 정보 (미등록 IP에 의한 API호출)
- 애플리케이션이름 : SOLN-이벗-A001
- 애플리케이션 아이디 : 30qwzUo7ED9I************ **(실제 이 아이디를 솔루션에서 사용하고 있습니다)**
- 미등록 IP에 의한 API호출-49.236.xxx.xxx

차단될 수 있다고 하던데 빠른 회신 부탁드려요.

감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

---
### ⚠️ 문의 내용 내 민감정보가 포함되어있습니다. ###
작성하신 포스트 원문 내 잠재적인 문제가 될 수 있는 민감정보가 포함되어있습니다. 해당 부분을 직접 확인하시어 내용을 수정해주시기 바랍니다.
본문 내용 수정 후 우측 상단의 수정 이력("edited" 글자)으로 민감정보가 포함된 수정 이전 내용이 표시되지 않도록 수정 이력도 함께 삭제해주시기 바랍니다.
 
- 대상정보: 이메일 주소 
---

운용중인 솔루션에 대한 'API호출 IP'를 지정하실 경우 등록된 IP로 호출된 건은 모니터링 시스템에 누적되지 않습니다.
솔루션 애플리케이션은 대상 커머스솔루션에 종속되기에 [[커머스API센터 > 내 솔루션](https://apicenter.commerce.naver.com/ko/member/my-solutions)]에서 대상 솔루션에 대한 API호출 IP를 지정해보시기 바랍니다.

**Comment by ebutmanager** [comment]
구조를 이해했습니다.
감사합니다.
